sds_data_manager/lambda_code/SDSCode/database/database.py

```diff
@@ -0,0 +1,5 @@
+"""Create engine that will be used to create schema or make query"""
+from SDSCode.database import config
+from sqlalchemy import create_engine
```

Comment:
```suggestion
from sqlalchemy import create_engine

from SDSCode.database import config
```

Reply:
again pre-commit undoes this change.

Reply:
Hmm, that is odd to me. Since one is a third-party import and the other is a local package import.

Reply:
yep